src/main/resources/db/migration/V6.5.0.6.sql

```diff
@@ -0,0 +1,2 @@
+-- Add this text will help to customized name in metadata source facet
+ALTER TABLE harvestingclient ADD COLUMN IF NOT EXISTS sourcename TEXT;
```

Comment:
We are close to releasing 6.6 and will be at code freeze (no more PRs merged) in a week on March 7th. With lots of merging going on I've already bumped the version of of this SQL script twice.

Mostly I want to communicate that with little time left, this PR might need to get bumped to 6.7. We'll see.